### PR TITLE
Add release onboarding issue template

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
**What type of PR is this?**

/kind documentation
/sig release

**What this PR does / why we need it**:

Add release onboarding issue template, list tasks checklist when a new release onboarding.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+title: "[RELEASE ONBOARDING] release"
+labels: sig/release
+
+---
+
+# New Release Onboarding
+
+This is an issue template aims to provide a comprehensive checklist template for releases onboarding. It includes a detailed list of preparation tasks to complete before the release and additional checkpoints to ensure everything runs smoothly after the release.
+
+## Preparations for release onboarding
```
Comment (fujitatomoya) [inline .github/ISSUE_TEMPLATE/release-onboarding.md:13]: nit; but it sounds like more requirement here.

```suggestion
## Requirements for release onboarding
```

**Review by fujitatomoya — APPROVED** [review]
i second this, it would be helpful for users to track the staging release.

File: .github/ISSUE_TEMPLATE/release-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] All CI tests pass.
+
+
+## After release onboarding
+
+- [ ] Draft staging repository release.
+- [ ] Update [Kubernetes compatibility](https://github.com/kubeedge/kubeedge?tab=readme-ov-file#kubernetes-compatibility).
+- [ ] Release blog.
+- [ ] Update KubeEdge [latest version](https://github.com/kubeedge/website/blob/master/functions/latestversion.js).
+- [ ] Update KubeEdge documentation version.
```
Comment (fisherxu) [inline .github/ISSUE_TEMPLATE/release-onboarding.md:24]: Update kubernetes version in github action CI

**Review by fisherxu — COMMENTED** [review]
/approve
cc @WillardHu @zhiyingfang2022 @fujitatomoya

**Pull request merged**